src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -3,15 +3,105 @@
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class GildedRoseTest {
 
     @Test
-    void foo() {
+    void qualityIsNeverNegative() {
         Item[] items = new Item[] { new Item("foo", 0, 0) };
         GildedRose app = new GildedRose(items);
         app.updateQuality();
-        assertEquals("fixme", app.items[0].name);
+        assertTrue(app.items[0].quality >= 0);
+    }
+
+    @Test
+    void qualityIsNeverMoreThan50() {
+        Item[] items = new Item[] { new Item("Aged Brie", 0, 50) };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertTrue(app.items[0].quality <= 50);
+    }
+
+    @Test
+    void qualityDegradesTwiceAsFastWhenPassedSellBy() {
+        Item[] items = new Item[] { new Item("FLAvorTown", 0, 50) };
```

Comment:
What is the value for `SellBy`?

Reply:
I assume it's `0`?

**Suggestion:** Might want to consider named params like:
```Java
new Item(name: "FLAvorTown", sellBy: 0, quantities: 50)
```

Reply:
Addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/12/commits/d638c0d71a1bf497cbb21dc0c3b8a78e9d2bbfde)

We noticed that the variable is set to `sellIn` within the `Item` class. So we did need to update the test names to reflect the proper variable of `sellIn` but in regards to your comment we do agree that specifying the unit would be more accurate. 

So we extracted the arguments into local variables that are now passed as arguments. And added "days" to the `sellIn` parameter. 

If you are not comfortable with the parameter name being `sellIn` we would need to go in and make a change to the `Item` class which is off limits at this point.

Reply:
I prefer refactoring it to use named params instead because it's much more explicit and has fewer indirections.

Reply:
Marking resolved as this is not a blocker. We can talk about this approach during our next IPM if that is ok with you.